src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -332,18 +386,24 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
 
         this.loadingProjectAssets = true;
 
-        const projectAssets = _.values(this.props.project.assets);
+        // Get all root project assets
+        const rootProjectAssets = _.values(this.props.project.assets)
```

Comment:
I would rename this `projectRootAssets` since it's looking for all the root assets in a project